services/AUTOMATIC1111/Dockerfile.rocm

```diff
@@ -0,0 +1,70 @@
+FROM alpine/git:2.36.2 as download
+
+COPY clone.sh /clone.sh
+
+
+RUN . /clone.sh stable-diffusion-stability-ai https://github.com/Stability-AI/stablediffusion.git cf1d67a6fd5ea1aa600c4df58e5b47da45f6bdbf \
+  && rm -rf assets data/**/*.png data/**/*.jpg data/**/*.gif
+
+RUN . /clone.sh CodeFormer https://github.com/sczhou/CodeFormer.git c5b4593074ba6214284d6acd5f1719b6c5d739af \
+  && rm -rf assets inputs
+
+RUN . /clone.sh BLIP https://github.com/salesforce/BLIP.git 48211a1594f1321b00f14c9f7a5b4813144b2fb9
+RUN . /clone.sh k-diffusion https://github.com/crowsonkb/k-diffusion.git ab527a9a6d347f364e3d185ba6d714e22d80cb3c
+RUN . /clone.sh clip-interrogator https://github.com/pharmapsychotic/clip-interrogator 2cf03aaf6e704197fd0dae7c7f96aa59cf1b11c9
+RUN . /clone.sh generative-models https://github.com/Stability-AI/generative-models 45c443b316737a4ab6e40413d7794a7f5657c19f
+
+
+FROM rocm/pytorch:rocm6.0.2_ubuntu22.04_py3.10_pytorch_2.1.2
```

Comment:
Is this the only difference between this docker file and the nvidia one?

Reply:
Additionally, all references to `CodeFormer` was removed, and the env variable `NVIDIA_VISIBLE_DEVICES`

Reply:
I think the environment variable would not make a difference in this case right?

I am wondering if we can configure this different image as a build arg instead of duplicating the entire dockerfile